lib/Doctrine/DBAL/Schema/Table.php

```diff
@@ -852,12 +852,8 @@ private function _createIndex(array $columns, string $indexName, bool $isUnique,
             throw IndexNameInvalid::new($indexName);
         }
 
-        foreach ($columns as $index => $value) {
-            if (is_string($index)) {
-                $columnName = $index;
-            } else {
-                $columnName = $value;
-            }
```

Comment:
FWIW, this might be used to support different formats of columns like and `['foo', 'bar' => ['option' => 'value']]`. Not sure if that's still the case but may be worth double-checking.

Reply:
Before deleting this code, I've double-checked manually that all calls pass an `array<int, string>`, so I don't think this is relevant anymore. And this doesn't break any tests, which makes me quite confident.